README.rst

```diff
@@ -20,122 +20,97 @@ README for Pylint - https://pylint.pycqa.org/
+oriented checks), black_ and isort_ (auto-formatting), autoflake_ (automated removal of
+unused import or variable), pyupgrade_ (automated upgrade to newer python syntax) and
+pydocstringformatter_ (automated pep257).
 
-Pull requests are amazing and most welcome.
+.. _flake8: https://gitlab.com/pycqa/flake8/
+.. _bandit: https://github.com/PyCQA/bandit
+.. _mypy: https://github.com/python/mypy
+.. _pyright: https://github.com/microsoft/pyright
+.. _pyre: https://github.com/facebook/pyre-check
+.. _black: https://github.com/psf/black
+.. _autoflake: https://github.com/myint/autoflake
+.. _pyupgrade: https://github.com/asottile/pyupgrade
+.. _pydocstringformatter: https://github.com/DanielNoord/pydocstringformatter
+.. _isort: https://pycqa.github.io/isort/
 
 Install
 -------
 
-Pylint can be simply installed by running::
+For command line use, pylint is installed with::
 
     pip install pylint
 
-If you are using Python 3.7.2+, upgrade to get full support for your version::
-
-    pip install pylint --upgrade
-
-If you want to install from a source distribution, extract the tarball and run
-the following command ::
-
-    python setup.py install
-
-
-Do make sure to do the same for astroid, which is used internally by pylint.
-
-For debian and rpm packages, use your usual tools according to your Linux distribution.
-
-More information about installation and available distribution format
-can be found here_.
-
-Documentation
--------------
-
-The documentation lives at https://pylint.pycqa.org/.
-
-Pylint is shipped with following additional commands:
-
-* pyreverse: an UML diagram generator
-* symilar: an independent similarities checker
-* epylint: Emacs and Flymake compatible Pylint
-
-
-Testing
--------
-
-You should be able to install our tests dependencies with::
-
-    pip install -r requirements_test.txt
+It can also be integrated in most editors or IDE. More information can be found
+`in the documentation`_.
 
-You can then use pytest_ directly. If you want to run tests on a specific portion of the
-code with pytest_ and your local python version::
+.. _in the documentation: https://pylint.pycqa.org/en/latest/user_guide/installation.html
 
-    # ( pip install pytest-cov )
-    python3 -m pytest
-    # Everything in tests/message with coverage for the relevant code:
-    python3 -m pytest tests/message/ --cov=pylint.message
-    coverage html
-    # Only the functional test "missing_kwoa_py3":
-    python3 -m pytest "tests/test_functional.py::test_functional[missing_kwoa_py3]"
+Contributing
+------------
 
-You can also *optionally* install tox_. To run the test suite for a particular
-Python version, with tox you can do::
+We welcome all contributions, doc, code, checking issues for duplicate or telling us
+that we can close them, confirming that it's still an issue, `creating issues because
+you found a bug or want a feature`_... everything helps !
```

Comment:
```suggestion
We welcome all forms of contributions such as updates for documentation, new code, checking issues for duplicates or telling us
that we can close them, confirming that issues still exist, `creating issues because
you found a bug or want a feature`_, etc. Everything is much appreciated!
```